src/streams.ts

```diff
@@ -0,0 +1,17 @@
+import convert from 'buffer-to-stream'
+
+export function bufferToStream (buffer: Buffer): any {
```

Comment:
What does this function add over the 'convert' function that was imported?

Reply:
mainly the consistent naming (`streamToBuffer`, `bufferToStream`). But I'm thinking maybe I should take out the whole streams thing and just use Buffers throughout, it would make the code a lot cleaner, and currently, there is no end-to-end streaming anyway (the reason to use streams in the first place was that it would allow use to pipe the read stream from the httpReq directly to the write stream in Redis, but that's currently impossible anyway because the read stream from the httpReq is just the body, and the write stream in Redis is body+contentType+Etag.

Reply:
Okay seems good for now then. I guess you'll refactor later if you think it's needed.